pkg/controller/controller_instance.go

```diff
@@ -203,10 +223,37 @@ func (c *controller) reconcileInstanceDelete(instance *v1alpha1.Instance) error
 	return nil
 }
 
+// isInstanceFailed returns whether the instance has a failed condition with
+// status true.
+func isInstanceFailed(instance *v1alpha1.Instance) bool {
```

Comment:
Yeah, the only way to resolve this (for now) is to delete and recreate.  Note that if you have something like 'I couldn't dial the broker', that type of error will get retried.  The only ones we consider terminal are errors where the broker has given back an http status code that indicates a failure.

I'd be open in the future to some mechanism (maybe a porcelain command) that makes such an instance get retried.  For now, we will not continually retry such instances.